scripts/src/services/payment.ts

```diff
@@ -45,14 +69,18 @@ export async function createWallet(walletAddress: string, appId: string, logger:
 	console.log("wallet creation took " + (performance.now() - t) + "ms");
 }
 
-export async function getWalletData(walletAddress: string, logger: LoggerInstance) {
-	// XXX missing definitions
+export async function getWalletData(walletAddress: string, logger: LoggerInstance): Promise<Wallet> {
 	const res = await axios.default.get(config.payment_service + "/wallets/" + walletAddress);
 	return res.data;
 }
 
-export async function getPaymentData(orderId: string, logger: LoggerInstance) {
-	// XXX missing definitions
-	const res = await axios.default.get(config.payment_service + "/orders/" + orderId);
+export async function getPaymentData(orderId: string, logger: LoggerInstance): Promise<Payment> {
+	const res = await axios.default.get(config.payment_service + "/payments/" + orderId);
```

Comment:
I would have used template literals but suite yourself.
Maybe move `/payments` into a const since your are using it in multiple locations.